### PR TITLE
Prefer native implementation over polyfill

### PR DESCRIPTION
**Pull request by jhnns**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ exports.active = function(item) {
 };
 
 // That's not how node.js implements it but the exposed api is the same.
-exports.setImmediate = function(fn) {
+exports.setImmediate = typeof setImmediate === "function"? setImmediate : function(fn) {
```
Comment (jryans) [inline main.js]: Nit: Use a space before the `?`

File: main.js [hunk 2]
```diff
@@ -49,6 +49,6 @@ exports.setImmediate = function(fn) {
   return id;
 };
 
-exports.clearImmediate = function(id) {
+exports.clearImmediate = typeof clearImmediate === "function"? clearImmediate : function(id) {
```
Comment (jryans) [inline main.js]: Nit: same as above

**Comment by jryans** [comment]
Okay, two tiny style nits, but I think this approach is fine. Seems like most browsers pre-bind `setImmediate`, so we can just use this simpler version.

**Comment by jhnns** [comment]
Spaces! :+1:

**Comment by jryans** [comment]
Published as `1.3.0`.

**Comment by jhnns** [comment]
N-n-n-n-ice!

**Pull request merged**